gnovm/stdlibs/std/emit_event_test.go

```diff
@@ -13,7 +13,11 @@ import (
 
 func TestEmit(t *testing.T) {
 	m := gno.NewMachine("emit", nil)
-	pkgPath := CurrentRealmPath(m)
+
+	elgs := sdk.NewEventLogger()
```

Comment:
seems we don't need this line.

Reply:
It needs a context check below. It will panic if it doesn't pass.

Reply:
I mean like this: 	
m.Context = ExecContext{}

the logger is not actually used by X_getRealm.